backend/de.metas.deliveryplanning.base/src/main/sql/postgresql/system/43-de.metas.deliveryplanning/5675361_sys_gh14881_create_sysConfig.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO AD_SysConfig (AD_Client_ID, AD_Org_ID, AD_SysConfig_ID, ConfigurationLevel, Created, CreatedBy, EntityType, IsActive, Name, Updated, UpdatedBy, Value) VALUES (0, 0, 541440, 'O', TO_TIMESTAMP('2023-02-03 01:23:45', 'YYYY-MM-DD HH24:MI:SS'), 100, 'D', 'Y', 'de.metas.DeliveryPlanning.import.formatId', TO_TIMESTAMP('2023-02-03 01:23:45', 'YYYY-MM-DD HH24:MI:SS'), 100, '540022')
```

Comment:
where is this sysconfig used?

Reply:
in `DeliveryPlanningDataFileAttachmentListener` where I'm trying to auto-import a file contents when the attachment is dropped onto the page.
I'm providing a default there, added the sysconfig more for visibility (not even sure if I should not hardcode it there and be done with it).

Reply:
loud thinking: why not hardcoding the InternalName and then use de.metas.impexp.DataImportService#getDataImportConfigByInternalName.... wdyt?